### ISSUE TITLE
Redmine-Git-Branch: Status message about having switched to a branch is displayed as Error

### ISSUE THREAD

**Issue by Pinolo** [sparkfabrik/sparktool#40]
Sample output:

```
Branch: "XMP-006_8921_find_the_droids_they_were_looking_for" created 
Error: Switched to a new branch 'feature/XMP-006_8921_find_the_droids_they_were_looking_for'
```

The problem seems to be with the attempt to run a command to push the new branch, after branch creation.

**Comment by paolomainardi** [comment]
@alessiopiazza this issue still exists ?

**Comment by alessiopiazza** [comment]
this issue should be fixed by #13
